ConveyorBot/marker_types_and_placing_them.md

```diff
@@ -27,10 +27,29 @@ The most important things to remember before setting up the route of markers are
 - GO marker should be used for one-way routes, where robot can drive only in one direction
+- Before sticking markers to the ground with their glue, it is recommended to paste them to the ground with a tape, so that you can test if your route setup is correct. After testing the route, stick them onto the ground normally.
 
 
 Markers are sticky stickers that can be peeled off and placed on another floor location. But it is not recommended to peel it off too many times as the glue on the marker can dry out.
 
 An example of a simple marker layout is shown in the figure below.
 
 <img src="ConveyorBot/assets/Map_example1.png" >
+
+## Circuit route
+
+If you want multiple robots to drive on the same marker setup, you should arrange the markers in a so called "Circuit route", so that the robots won't collide into each other. This type of marker setup is recommended also if you have only one robot, because you never know when you will add another robot to your fleet. The Circuit route has an inbound and an outbound path. Inbound serves as the path for robots to drive to the goal and the outbound is the path which is used for the robots to return back to the starting point. The inbound and outbound paths should be at least 1 meter apart (TODO: determine the exact minimum distance), so that the robots won't obstruct each other. On both ends of the path, markers should be arranged in a specific way, shown in the bellow image.
+When setting up a route, the measurements from the bellow image need to be considered (TODO: test extensively if the measurements in the image are actually correct and update the values if necessary).
+If you want to include a STOP marker into the Circuit route, the best way of placing it is one meter behind the GO marker. You don't place the STOP marker 4 meters away from GO marker, because you might want to ignore the STOP marker in the case if you don't want the robot to stop on it. This would lead to 8 meters distance between GO markers, which is too far apart, if the STOP marker would be 4 meters away from previous and following GO marker.
+As seen from the image, on each end of the Circuit route, there is a TURN marker, which you need to include into the route on the touchscreen. This can't be a GO marker, because it would be ignored as it is turned in opposite direction as the robot's driving direction.
+
+<img src="ConveyorBot/assets/Map_example_circuit.png" >
+
+### Branches from main circuit route
+
+TODO: Desribe this after collision avoidance when returning on the main route from a branch is done.
+
+## Ignored opposite markers
+
+By default, the GO and STOP markers which are rotated in the opposite direction as the robot is driving are ignored. This is required for the Circuit route to work correctly when inbound and outbound paths are close together, becasue otherwise, if the robot is driving on the inbound path, it might detect and follow the opposing markers from the outbound path.
```

Comment:
"This is required for the Circuit route to work correctly when inbound and outbound paths are close together, becasue otherwise, if the robot is driving on the inbound path, it might detect and follow the opposing markers from the outbound path." -> Separating this into two or 3 different sentences would help with clarity.